docs/data/about/teamMembers.json

```
@@ -40,12 +40,12 @@
     "title": "Software Engineer - Core",
     "location": "Silesia, Poland",
     "locationCountry": "pl",
-    "about": "Motorcyclist, gamer, and coder (UI and more!)",
+    "about": "Motorcyclist, gamer, and coder (UI and more!).",
     "twitter": "michaldudak",
     "github": "michaldudak"
   },
   {
-    "name": "Siriwat Kunaporn",
+    "name": "Jun Kunaporn",
```

```suggestion
    "name": "Siriwat Kunaporn",
```
Or we should rename the image to jun-kunaporn.png.